scripts/integration_parakeet.ps1

```
@@ -0,0 +1,68 @@
+$deadline = (Get-Date).AddSeconds($HealthTimeoutSeconds)
+$healthy = $false
+while ((Get-Date) -lt $deadline) {
+    try {
+        $r = Invoke-WebRequest -Uri 'http://localhost:5092/health' -TimeoutSec 5 -UseBasicParsing -ErrorAction Stop
+        if ($r.StatusCode -eq 200 -and $r.Content -match '"status"\s*:\s*"ok"') {
+            $healthy = $true
+            break
+        }
+    } catch {
+        # fall through to unconditional sleep below
+    }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)